from django import ValidationError

class UserValidations:
    
    def _init__(self,user):
        self.user = user
    
    def validations(self):
        pass